### PR TITLE
[DOCU-3181] (MinK) Updating existing Konnect topics: Labels and Compatibility tables

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

What did you change and why?
- Added a compatibility table to the Compatibility topic in Konnect with info about which Mesh version supports MinK
- Added a section and a link to the Kubernetes labels Mesh topic in the Labels reference page

Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc.
DOCU-3181

### Testing instructions

Netlify link: 

### Checklist 

- [ ] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/compatibility.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ data plane.
...
 
 
+## {{site.mesh_product_name}} compatibility
+
+To use [Mesh Manager](/konnect/mesh-manager/), you must also use a compatible version of {{site.mesh_product_name}}. 
+
+|                                  | {{site.konnect_saas}} | First supported patch version 
+|--------------------------------|:---------------------:|-----------------------------
+| {{site.mesh_product_name}} 2.2.x | <i class="fa fa-check"></i> | -- |
+| {{site.mesh_product_name}} 2.1.x | <i class="fa fa-times"></i> | N/A |
```
Comment (lena-larionova) [inline app/konnect/compatibility.md:37]:
```suggestion
| {{site.mesh_product_name}} 2.1.x or earlier | <i class="fa fa-times"></i> | N/A |
```

**Review by lena-larionova — APPROVED** [review]
Some minor edits, otherwise LGTM.

File: app/konnect/compatibility.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ data plane.
 | {{site.ee_product_name}} 2.4.x or earlier | <i class="fa fa-times"></i>    | --
 
 
+## {{site.mesh_product_name}} compatibility
+
+To use [Mesh Manager](/konnect/mesh-manager/), you must also use a compatible version of {{site.mesh_product_name}}. 
+
+|                                  | {{site.konnect_saas}} | First supported patch version 
+|--------------------------------|:---------------------:|-----------------------------
+| {{site.mesh_product_name}} 2.2.x | <i class="fa fa-check"></i> | -- |
```
Comment (lena-larionova) [inline app/konnect/compatibility.md:36]:
```suggestion
| {{site.mesh_product_name}} 2.2.x | <i class="fa fa-check"></i> | -- |
```
Which version is supported?

File: app/konnect/compatibility.md [hunk 3]
```diff
@@ -27,6 +27,15 @@ data plane.
 | {{site.ee_product_name}} 2.4.x or earlier | <i class="fa fa-times"></i>    | --
 
 
+## {{site.mesh_product_name}} compatibility
+
+To use [Mesh Manager](/konnect/mesh-manager/), you must also use a compatible version of {{site.mesh_product_name}}. 
+
+|                                  | {{site.konnect_saas}} |
+|--------------------------------|:---------------------:|
+| {{site.mesh_product_name}} 2.3.x | <i class="fa fa-check"></i> |
```
Comment (cloudjumpercat) [inline app/konnect/compatibility.md:36]: ❗ Reviewers: Charly said that versions later than 2.2.x would be supported. 2.3.x will be the next minor version, but a patch version could come out for 2.2.x. Is there a better way to represent this than what I did?

Comment (lena-larionova) [reply app/konnect/compatibility.md:36]: I would suggest adding a "First supported patch version" column here, same as the table above. For 2.3.x, that might mean 2.3.0, and that's fine. When the supported 2.2.x patch comes out, we can add an entry for it here as well.

**Pull request merged**